src/components/Header/Navigation.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Navigation.scss";
+import * as ROUTES from "../../constants/routes";
+
+export default function Navigation() {
+  return (
+    <ul className="menu-bar">
+      <li style={{ fontWeight: "40px" }}>
+        <Link
+          style={{ textDecoration: "none", color: "black", fontWeight: "40px" }}
```

Comment:
Avoid inline style, please use className & declare them in corresponding  `.scss` file